client/remote.cpp

```diff
@@ -404,7 +404,7 @@ static int build_remote_int(CompileJob &job, UseCSMsg *usecs, MsgChannel *local_
             struct stat buf;
 
             if (stat(version_file.c_str(), &buf)) {
-                log_perror("error stat'ing version file");
```

Comment:
I still want the log_perror.  Can we overload log_perror to return a stream that additional information can be sent to?